edx_adapt/api/resources/course_resources.py

```diff
@@ -90,6 +91,13 @@ def post(self, course_id):
         try:
             self.repo.enroll_user(course_id, args['user_id'])
             first_prob = self.selector.choose_first_problem(course_id, args['user_id'])
+            logger.info("Pre-assessment problem is found and selected: {}".format(first_prob))
+            if not first_prob:
+                first_prob = self.selector.choose_next_problem(course_id, args['user_id'])
+                logger.info(
```

Comment:
This log message right after the previous one is going to confuse reader.
Please restructure to show only one message.